src/main/java/at/hannibal2/skyhanni/test/TestCopyChestData.kt

```diff
@@ -0,0 +1,80 @@
+@file:Suppress("SENSELESS_COMPARISON") // intellij does a little trolling and assumes all ItemStacks will never be null. this suppresses such warnings.
```

Comment:
Always apply suppressions as locally as possible. I also haven't had that issue with intellij before. Maybe check that warning once again.

Reply:
>Maybe check that warning once again.

already did, and that's how i get hard crashes due to `NPE`s (yep, even skipped skyhanni's usual "click to copy error" chat message)

Reply:
proof: <img src="https://github.com/hannibal002/SkyHanni/assets/51521765/0b649ed8-c376-4fef-b9d8-e9da7ed825e2">

i could just do `?.` calls, but i added a config option to include data on null slots. may as well make use of the `== null` check at that point imo

Reply:
The issue in this case is that you use an `Array<ItemStack>` as type, when really it should be a `Array<ItemStack?>`. The lint works correctly, your types are just wrong. Suppressing that lint here just means that you will not get proper type checking if you access the stack in another place. We should not allow implicit nulls anywhere.

Reply:
👍